Extension/gulpfile.js

```diff
@@ -192,7 +192,7 @@ const processHtmlFiles = () => {
 const traverseJson = (jsonTree, descriptionCallback, prefixPath) => {
     for (let fieldName in jsonTree) {
         if (jsonTree[fieldName] !== null) {
-            if (typeof (jsonTree[fieldName]) == "string" && fieldName === "description") {
+            if (typeof (jsonTree[fieldName]) == "string" && (fieldName === "description" || fieldName === "markdownDescription")) {
```

Comment:
It might be a good idea to automatically add a hint whenever a `markdownDescription` is detected, indicating that the string could contain markdown and the translator should avoid breaking it.  Since the hints are stored in an array, you can add to it instead of replacing one if existing.  Or, alternatively, you could add a `descriptionHint` field to each `markdownDescription` in the json schema file.

Reply:
These changes rely on the translators not touching text inside `` unless otherwise directed to by a hint. I think we just need to make sure that is communicated to the localization team if they don't already know.

Reply:
I suspect there are quite a few people involved in localization - multiple people per language.  It would be a good idea to add hints whenever possible.  They are likely not highly technical people and keeping them trained to recognize requirements like these would seem to be an ongoing issue.

Reply:
I just added the hints explicitly instead of via code, so that's it's more visible.